packages/client/src/links/wsLink/wsClient/wsClient.ts

```
@@ -0,0 +1,431 @@
+  /**
+   * Sends a message or batch of messages directly to the server.
+   */
+  private send(
+    messageOrMessages: TRPCClientOutgoingMessage | TRPCClientOutgoingMessage[],
+  ) {
+    if (!this.activeConnection.isOpen()) {
+      throw new Error('Active connection is not open');
+    }
+
+    const messages =
+      messageOrMessages instanceof Array
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)